src/sparql/QueryObjectBuilder.ts

```diff
@@ -0,0 +1,48 @@
+import allNamespaces from '@/sparql/RdfNamespaces';
+
+export default class QueryObjectBuilder {
+	// TODO: Change the type once #27 is merged
+	private queryObject: any;
+
+	public constructor() {
+		this.queryObject = {
+			queryType: 'SELECT',
+			variables: [],
+			where: [],
+			type: 'query',
+			prefixes: allNamespaces
+		};
+	}
+
+	public buildFromQueryRepresentation( queryRepresentation: any ) {
```

Comment:
maybe this can be changed already? was checking the types and not sure which it corresponds to

Reply:
Done.